### PR TITLE
Improve formatting!

### PR DESCRIPTION
**Pull request by WizzyGeek**
Improve code and readme!

(Enjoy the 1st PR to your repo! :D)

### CODE REVIEW

File: game.py [hunk 1]
```diff
@@ -31,7 +31,20 @@
...
+     Smokes: int = 0,
+     Place: str = "Entrance",
+     Screw: bool = False,
+     Key: bool = False,
+     RedKey: bool = False,
+     GreenKey: bool = False,
+     Firester: bool = False,
+     Gryphon: bool = False,
+     WoodenSword: bool = False,
+     GoldenSword: bool = False) -> None:
```
Comment (knockonwoodgrain) [inline game.py:47]: Why would i do this?

File: game.py [hunk 2]
```diff
@@ -570,6 +582,7 @@ def UnidentifiedInput(self):
                 print("put take smokes")
             elif self.Place == "Gryphon":
                 print("do ou wanna fight him or not??")
+
 if __name__ == "__main__": 
-  starts = Player(100, 0, 0, False, False, False, False)
-  starts.CheckpointLoad()
\ No newline at end of file
+  starts = Player()
+  starts.CheckpointLoad()
```
Comment (knockonwoodgrain) [inline game.py:588]: does this even work?

File: game.py (hunk 1 above)
Comment (WizzyGeek) [reply game.py:47]: PEP 8 recommends a line width limit of 70

File: game.py (hunk 2 above)
Comment (WizzyGeek) [reply game.py:588]: Yes, I switched it to default arguments specified in `__init__` you might wanna make `Player.CheckpointLoad` a class method though

**Comment by knockonwoodgrain** [comment]
thanks dude

**Comment by WizzyGeek** [comment]
No problem 👍

**Pull request merged**